.gitignore

```diff
@@ -7,6 +7,7 @@ _input/
 test/user.env
 user.env
 test/acs-engine-test/acs-engine-test
-
-test/acs-engine-test/acs-engine-test.exe
-pkg/operations/junit.xml
\ No newline at end of file
+
+test/acs-engine-test/acs-engine-test.exe
+pkg/operations/junit.xml
```

Comment:
Yes, both `go test` under the kubernetesupgrade folder or `go test ./...` both generates the file.